.fixtures.yml

```diff
@@ -0,0 +1,17 @@
+# This file can be used to install module dependencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+
+---
+fixtures:
+  repositories:
+    stdlib:
+      repo: git://github.com/puppetlabs/puppetlabs-stdlib.git
+      ref: 4.20.0
```

Comment:
we're starting off fresh, so use a higher version of stdlib. v6 supports back to Puppet 5.5

Reply:
Fixed.